olclient/schemata/import.schema.json

```
@@ -159,6 +163,17 @@
 	  "examples": ["duc d'Otrante"]
 	}
       }
+    },
+    "contributors": {
```

Rename this to match the other one. Also singular since this defines a singly type.
```suggestion
    "import_contributor": {
```